cmd/admin-trace.go

```diff
@@ -42,23 +42,27 @@ var adminTraceFlags = []cli.Flag{
 	},
 	cli.BoolFlag{
 		Name:  "all, a",
-		Usage: "trace all traffic (including internode traffic between MinIO servers)",
+		Usage: "trace all traffic (includes internode traffic from MinIO cluster)",
 	},
-	cli.StringFlag{
+	cli.IntSliceFlag{
 		Name:  "status-codes",
```

Comment:
Maybe it makes sense to call this 'status-code' instead of 'status-codes', same for other applicable params

Reply:
done @vadmeste